File: src/easy/future_people.zig
const expectEqual = @import("std").testing.expectEqual;

pub fn futurePeople(population: i32, births: i32) i32 {
    return population + 360 * births;
}
test "Calculate population after 30 years" {
    try expectEqual(futurePeople(256, 2), 976);
    try expectEqual(futurePeople(3248, 6), 5408);
    try expectEqual(futurePeople(5240, 3), 6320);
}

File: src/easy/future_people.rs
pub fn future_people(population: u16, births: u16) -> u16 {
    population + 360 * births
}
#[cfg(test)]
mod tests {
    use super::future_people;
    #[test]
    fn test_future_people() {
        assert_eq!(future_people(256, 2), 976);
        assert_eq!(future_people(3_248, 6), 5_408);
        assert_eq!(future_people(5_240, 3), 6_320);
    }
}
